reproducible-build/Dockerfile

```diff
@@ -1,7 +1,8 @@
 FROM crops/poky@sha256:f51ae3279f98768514273061336421e686e13d0a42fdb056c0b88c9afeec8c56 as builder
 
 USER root
-RUN apt install -y repo
+RUN apt update && apt install -y repo=2.17.3-3 libssl-dev=3.0.2-0ubuntu1.18
```

Comment:
having apt update inside the dockerfile might cause the env to change. Hence, potentially impact the reproducibility of the measurement of the built images.
That's why I was avoiding apt update and directly installing dependencies with their latest version in that particular container image instead of fetching the up-to-date versions of such libraries with apt update

Reply:
`libssl-dev` is sadly not available, hence the update + freezing versions. There's eactly 5 packages installed, and all of those should be frozen - I'll find out which ones exactly

Reply:
That's unfortunate but maybe if we could provide the needed dependencies as statically linked precompiled would probably be better than risking the image to be not reproducible in the future